Content.Server/Anomaly/Components/AnomalousParticleComponent.cs

```diff
@@ -49,4 +49,13 @@ public sealed class AnomalousParticleComponent : Component
     /// </summary>
     [DataField("stabilityPerWeakeningeHit")]
     public float StabilityPerWeakeningeHit = -0.1f;
+
+    [DataField("AnomalyStabilizer")]
+    public bool AnomalyStabilizer = false;
```

Comment:
this should be `AnomalyDestabilizer`, considering that's what it does,

also, the naming for these three variables could be more concise. Perhaps:
- DestabilzingOverride
- WeakeningOverride
- SeverityOverride

Reply:
also these all should be documented. Just explain that each one overrides the behavior for having the specific particle type.

Reply:
added little description for each.